### ISSUE TITLE
Make it work for 10.12.x ?

### ISSUE THREAD

**Issue by hquick81** [guyaudio3/M-Audio-Firewire#1]
Hi, any chance to make this work for 10.12.x ?

**Comment by royip** [comment]
Joining this request.. Please don't let this Product die!

**Comment by estudiooka** [comment]
Sierra Please

**Comment by ServeurIMFP** [comment]
Yes please. I just fell on this thread and I was so hopeful. I tried it on Sierra but no go unfortunately :(
The app and the control panels work, but they don't see the interface. Pity…

**Comment by beyawnko** [comment]
same as the guy above. software doesn't see the interface, but the panels work and os reads it as 1814 boot loader. any thing i can experiment with?

**Comment by mjt79** [comment]
any progress getting this to work with Sierra?

**Comment by joeygl** [comment]
Sierra Please too! :(

**Comment by gumaerc** [comment]
@guyaudio3 Could you please post the source code to this somewhere so I can make an attempt to get this to work on modern operating systems?

**Comment by LucianoV78** [comment]
@gumaerc That's the point and hoping you're talking about Catalina.
I'm not very helpful but I can morally support :)

**Comment by gumaerc** [comment]
@LucianoV78 I'm using nvidia hardware so I'm stuck on 10.13 unfortunately.  Either way, I'm not going to be able to do anything unless OP responds.

**Comment by LucianoV78** [comment]
@gumaerc I understand

**Comment by kuklamac** [comment]
I would love to see an update for 12.x Mac Os x.
Thanks...

**Comment by gumaerc** [comment]
I doubt it's going to happen guys. I've taken the route of setting up a separate vlanned hackintosh that will permanently run 10.11 plugged into my ProjectMix, then I share the control + external midi devices over the network using Audio MIDI setup. It's kind of a pain in the ass and I would obviously much rather just plug in the firewire cable and have it work. Honestly though, it is kind of handy to be able to have any PC be able to connect to the control surface over the network. If you use the ProjectMix and the control surface functionality is what's important to you, I recommend you look into it. You can use a cheap old used mac mini to do the job.
